dynawo/sources/Models/Modelica/Dynawo/Electrical/Wind/IEC/BaseClasses/BaseWPP.mo

```diff
@@ -14,6 +14,8 @@ within Dynawo.Electrical.Wind.IEC.BaseClasses;
 
 model BaseWPP "Base model for Wind Power Plants from IEC 61400-27-1 standard"
   extends Dynawo.Electrical.Controls.IEC.Parameters.GridProtectionParameters;
+  extends Dynawo.Electrical.Controls.IEC.Parameters.PControlParameters;
```

Comment:
No need to extend these parameters since they were already extended in baseWPPControl

Reply:
if these parameters are not declared here in the top level model, they can't be modified when using the model with OM or Dynawo. This has been previously done for the grid protection combitables as we can see here on line 16 : GridProtectionParameters are declared in BaseWPP but also in BaseWT4 and in BaseGridProtection.
It may be part of another PR if you prefer.

Reply:
Okay, you are right.

We can keep them